src/utils/dom.ts

```diff
@@ -35,23 +53,50 @@ export function getModeName(el: HTMLElement): "light" | "dark" {
   return closestElWithMode?.classList.contains("calcite-mode-dark") ? "dark" : "light";
 }
 
+/**
+ * This helper returns the direction of a HTML element.
+ *
+ * @param {HTMLElement} el An element.
+ * @returns {Direction} The direction.
+ */
 export function getElementDir(el: HTMLElement): Direction {
   const prop = "dir";
   const selector = `[${prop}]`;
   const closest = closestElementCrossShadowBoundary(el, selector);
   return closest ? (closest.getAttribute(prop) as Direction) : "ltr";
 }
 
-export function getElementProp(el: Element, prop: string, fallbackValue: any): any {
-  const selector = `[${prop}]`;
+/**
+ * This helper returns the direction of a HTML element.
+ *
+ * @param {HTMLElement} el An element.
+ * @param {string} attribute An attribute name.
+ * @param {any} fallbackValue A fallback value.
+ * @returns {any} The value.
+ * @depecated
```

Comment:
deprecating this function as it shoudn't be used.

It actually doesn't return a prop, it returns an attribute.